app/views/case_workers/claims/show.html.haml

```diff
@@ -5,6 +5,7 @@
   = render 'errors/error_summary', ep: @claim, error_summary: t('.error_summary')
   = render 'errors/injection_error_summary', ep: claim
   = render 'warnings/conference_and_views_summary', claim: claim
+  = render 'warnings/scheme_12_fees_summary', claim: claim
```

Comment:
is there any way we can make this more generic

i.e. replace
```  
  = render 'warnings/conference_and_views_summary', claim: claim
  = render 'warnings/scheme_12_fees_summary', claim: claim
```
with
```
  = render 'warnings/injection_warning_summary', claim: claim
``

and then combine the two above into one

Reply:
👍 have replaced warnings/conference_and_views_summary and warnings/scheme_12_fees_summary with warnings/injection_warning_summary.